examples/log/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
```

per https://github.com/gazebosim/gz-cmake/issues/350

```suggestion
cmake_minimum_required(VERSION 3.22.1 FATAL_ERROR)
```